packages/shared/src/components/post/PostContentContainer.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement, ReactNode } from 'react';
+import classed from '../../lib/classed';
+
+interface PostContentContainerProps {
```

Comment:
I dont think we should have the option to hide articles from the squad here (you can create a task and skip this if it is not related to the modal)

<img width="648" alt="SCR-20230130-exc" src="https://user-images.githubusercontent.com/1488164/215417364-a2f198a8-6155-4a79-b1d7-b8e149b7f6f5.png">

Reply:
I already discussed this and product does not want to change any behaviour here at closed beta.